ehr_billing/src/org/labkey/ehr_billing/notification/BillingNotification.java

```diff
@@ -342,7 +342,7 @@ public void exec(ResultSet object) throws SQLException
                     projectDisplay = "None";
                 }
 
-                String financialAnalyst = rs.getString(FieldKey.fromString("investigatorLastName"));
+                String financialAnalyst = rs.getString(FieldKey.fromString("investigator"));
```

Comment:
@labkey-bpatel  and @labkey-ankurj can you confirm this is OK for both WNPRC and TNPRC?

Reply:
I am not sure if I've came across/tested runNotification functionality in TNPRC, but a text search for investigatorLastName in both of tnprc repositories has no results.

Reply:
Looks like there RequestAdminNotification in tnprc_ehr and ehrModules/EnterDataPanel.js has the key `org.labkey.onprc_ehr.notification.RequestAdminNotification`

Reply:
WNPRC has been using this in production, so looks like that change is fine.
Thanks @labkey-ankurj for checking in tnprc.